docs/quickstart.rst

```diff
@@ -113,3 +113,18 @@ option:
 
    If you have a ``.yamllint`` file in your working directory, it will be
    automatically loaded as configuration by yamllint.
+
+Use ``--list-files`` to list files to lint. This is useful to determine which
```

Comment:
Not sure if this is the best place to talk about `--list-files` :shrug: but I wasn't able to find a better one. Any hints much appreciated :nerd_face: 

Also, we could link `yaml-files` and `ignore` to configuration page. We could also mention `--list-files` when talking about `ignore` and `yaml-files` options :thinking:

Reply:
I agree that it isn't obvious what's the best place to talk about this new option...

After looking at [the current documentation structure](https://yamllint.readthedocs.io/en/stable/index.html), I believe the best place is **along the `ignore` and `yaml-files` sections**.

Here is the diff I propose. What do you think?
```diff
--- a/docs/configuration.rst
+++ b/docs/configuration.rst
@@ -136,6 +136,13 @@ directories, set ``yaml-files`` configuration option. The default is:
 The same rules as for ignoring paths apply (``.gitignore``-style path pattern,
 see below).
 
+If you need to know the exact list of files that yamllint would process, without
+really linting them, you can use ``--list-files``:
+
+.. code:: bash
+
+ yamllint --list-files .
+
 Ignoring paths
 --------------
 
@@ -204,6 +211,13 @@ or:
 
 .. note:: However, this is mutually exclusive with the ``ignore`` key.
 
+If you need to know the exact list of files that yamllint would process, without
+really linting them, you can use ``--list-files``:
+
+.. code:: bash
+
+ yamllint --list-files .
+
 Setting the locale
 ------------------
 
```

(I removed the "The output will look like" example)

Reply:
Sounds good to me :+1: 

Thanks for the patch and rewording :bow: :heart: